### DISCUSSION TITLE
Windows only project || Desktop project only

### DISCUSSION THREAD

**Discussion by Inerska** [unoplatform/uno#11437] [Q&A]
Hello looking for a template or the cli command to create a windows project only or if I can't, a desktop only project.

I've concluded with something like that : dotnet new unoapp -wasm=false -W=true -mobile=false -sw=false -sg=false -slf=false -vs=false -wpm=false -o .

But looks like that all projects ain't loading correctly and does not even work.
![image](https://user-images.githubusercontent.com/50674490/220965770-bf24dbdb-fe76-4fa7-ae35-bdaba9ee664b.png)

Thanks

**Comment by jeromelaban** [comment]
Thanks for the report. We're not getting such errors with the same command you provided.

Can you ensure that you're running the latest templates?

```
dotnet new uninstall Uno.ProjectTemplates.Dotnet
dotnet new install Uno.ProjectTemplates.Dotnet
```

**Comment by Inerska** [comment]
![image](https://user-images.githubusercontent.com/50674490/221139313-017afc2c-3c0a-4325-9299-c5398267ad65.png)
I got this project structture,

And this CLI output :
```
dotnet new unoapp -wasm=false -W=true -mobile=false -sw=false -sg=false -slf=false -vs=false -wpm=false -o .
The "Multi-Platform App (Windows App SDK)" template has been created successfully.

Processing post-creation actions... Please wait.
Restoring C:\Users\Inerska\test\test.Server\test.Server.csproj:
Failed to restore: File 'C:\Users\Inerska\test\test.Server\test.Server.csproj' not found.
Restoring C:\Users\Inerska\test\test.Windows\test.Windows.csproj:
  Determining projects to restore...
  Restored C:\Users\Inerska\test\test\test.csproj (in 505 ms).
  Restored C:\Users\Inerska\test\test.Windows\test.Windows.csproj (in 523 ms).
Restoration succeeded.
Post-action failed.
Manual instructions: Run 'dotnet restore'.
```

On rider got this : 
![image](https://user-images.githubusercontent.com/50674490/221139749-9afa4424-a5a5-467c-99b4-a482be004211.png)

And when I try to run `test.Windows` it does not even run. Same for `test` project.
By the way where should I write code inside ?

**Reply by jeromelaban** [reply]
Thanks. What do you mean by "it does not even run" ? If it's a build failure, can you [create a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html#generating-msbuild-binary-log-files)?

As for understanding how the solution is structure, [take a look at this documentation](https://platform.uno/docs/articles/uno-app-solution-structure.html).

Also, to understand, why do you need to create a windows only project?

**Reply by Inerska** [reply]
I just wanna try Uno Platform for a basic Windows-only platform project, with the Figma plugin.
